types/ember/test/create-negative.ts

```diff
@@ -0,0 +1,12 @@
+import { assertType } from './lib/assert';
+import Ember from 'ember';
+import { PersonWithNumberName, Person } from './create';
+
+Person.create({ firstName: 99 }); // $ExpectError
+Person.create({}, { firstName: 99 }); // $ExpectError
+Person.create({}, {}, { firstName: 99 }); // $ExpectError
+
+const p4 = new PersonWithNumberName();
+// assertType<number>(p4.get('fullName'));
+
+// assertType<Ember.ComputedProperty<string, string>>(p4.fullName); // $ExpectError
```

Comment:
Is there a story on these commented type assertions? Based on the assertions in `test/create.ts` I'd expect `p4.get('fullName')` and `p4.fullName` both to have type `number`, but I'm not sure what the aim is here.

Reply:
The line you're commenting on is most interesting. Currently the type of fullName is `string && ComputedProperty<string, string>`. This is not accurate. It should just be `string`. We'll resolve this when we apply the change described [here](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/28282#discussion_r213483610)